### PR TITLE
Finished book_descriptions script

### PR DESCRIPTION
**Pull request by Mephiles**
Original code by: @Sashank999. Item descriptions: Allo.

### CODE REVIEW

File: torntools/scripts/content/items/ttItems.js [hunk 1]
```diff
@@ -1,4 +1,50 @@
 const USABLE_ITEM_TYPES = ["Medical", "Drug", "Energy Drink", "Alcohol", "Candy", "Booster"];
+const BOOK_DESCRIPTIONS = {
+	744: "Incr. Str by 5% up to 10m upon completion.",
+	745: "Incr. Spe by 5% up to 10m upon completion.",
```
Comment (DeKleineKobini) [inline torntools/scripts/content/items/ttItems.js:4]: Isn't 'Spd' the more known short form?

File: torntools/scripts/content/items/ttItems.js [hunk 2]
```diff
@@ -1,4 +1,50 @@
...
+	746: "Incr. Def by 5% up to 10m upon completion.",
+	747: "Incr. Dex by 5% up to 10m upon completion.",
+	748: "Incr. all working stats by 5% up to 2.5k each upon completion.",
+	749: "Incr. blacklist & friend list by 100 upon completion.",
+	750: "Provides a free merit reset upon completion.",
+	751: "Removes a large amount of drug addiction upon completion.",
+	752: "Provides a passive 25% bonus to all stats (31 days).",
+	753: "Provides a passive 100% bonus to Str (31 days).",
+	754: "Provides a passive 100% bonus to Def (31 days).",
+	755: "Provides a passive 100% bonus to Spe (31 days).",
```
Comment (DeKleineKobini) [inline torntools/scripts/content/items/ttItems.js:14]: 'Spd'?

File: torntools/scripts/content/items/ttItems.js [hunk 3]
```diff
@@ -1,4 +1,50 @@
...
+	756: "Provides a passive 100% bonus to Dex (31 days).",
+	757: "Incr. all gym gains by 20% (31 days).",
+	758: "Incr. Str gym gains by 30% (31 days).",
+	759: "Incr. Def gym gains by 30% (31 days).",
+	760: "Incr. Spe gym gains by 30% (31 days).",
```
Comment (DeKleineKobini) [inline torntools/scripts/content/items/ttItems.js:19]: 'Spd'?

**Review by DeKleineKobini — APPROVED** [review]

**Comment by Sashank999** [comment]
But this PR should be closed as branches have been merged...

**Comment by DeKleineKobini** [comment]
It's not been merged yet, I merged master into this branch, to solve the merge conflicts. There is still a comment left.

**Pull request merged**